### PR TITLE
7xbid adapter

### PR DESCRIPTION
**Pull request by 7XBID00**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter 
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
[
    {
      code: 'test',
      mediaTypes: {
        banner: {
          sizes: [[300, 250], [300,600]],
        }
      },
      bids: [
        {
          bidder: '7xbid',
          params: {
            placementId: 1425292,
            currency: 'USD'

          }
        }
      ]
    },
    {
      code: 'test',
      mediaTypes: {
        native: {
          title: {
            required: true,
            len: 80
          },
          image: {
            required: true,
            sizes: [150, 50]
          },
          sponsoredBy: {
            required: true
          }
        }
      },
      bids: [
        {
          bidder: '7xbid',
          params: {
            placementId: 1429695,
            currency: 'USD'
          }
        },
      ],
    }
  ]
```
- contact email of the adapter’s maintainer
- [x] official adapter submission

### CODE REVIEW

**Comment by longkeng** [comment]
@robertrmartinez Can you check my PR and merge it to master if not any problem. 
Thank you!

**Comment by robertrmartinez** [comment]
Hi @longkeng Sorry for the delay but I will get this reviewed now!

We will get it into the Prebid.js release scheduled for tomorrow!

File: modules/7xbidBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      let bidResponse = {
+        requestId: bid.requestid,
+        cpm: bid.price,
+        creativeId: bid.creativeId,
+        currency: bid.cur,
+        netRevenue: NET_REVENUE,
+        ttl: 700
+      };
+
+      if (bid.hasOwnProperty('title')) { // it is native ad response
```
Comment (robertrmartinez) [inline modules/7xbidBidAdapter.js]: Guessing you trust that your ad server will only respond with native if the request was native, so no check needed for the bidRequest being native here?

File: modules/7xbidBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        data.apiv = '1.1.0'
+      }
+
+      if (refererInfo && refererInfo.referer) {
+        data.referer = refererInfo.referer;
+      } else {
+        data.referer = '';
+      }
+
+      serverRequests.push({
```
Comment (robertrmartinez) [inline modules/7xbidBidAdapter.js]: There was a push to have all adapters support SRA (Single Request Architecture).

I am unsure if this is till a requirement for new / existing adapters, so I have contacted the rest of the Prebid.org team and asked.

I will get back to you as soon as I get an answer, but wanted to give you a heads up!

Comment (robertrmartinez) [reply modules/7xbidBidAdapter.js]: Ok got an answer and we will not require it.
But a future requirement may come up where we do, just for your information!

That change would have comms ahead of time and we would make it known to all adapters which need to be updated well ahead of time!

**Comment by bretg** [comment]
This needs a docs PR. See https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/adkernel.md as an example.

**Comment by robertrmartinez** [comment]
@7XBID00 ^^ I forgot to make sure you had docs open.

Please get a PR as soon as you can

File: modules/7xbidBidAdapter.js (hunk 1 above)
Comment (7XBID00) [reply modules/7xbidBidAdapter.js]: only native bid response has `title` field so that I use it to check

**Comment by 7XBID00** [comment]
@bretg @robertrmartinez  Hi, I created PR for dev doc in here https://github.com/prebid/prebid.github.io/pull/1583 , please take a look

**Pull request merged**